posydon/binary_evol/DT/gravitational_radiation/default_gravrad.py

```diff
@@ -2,8 +2,8 @@
 
 def default_gravrad(a, e, primary, secondary, verbose = False):
 
-    m1 = primary["mass"]
-    m2 = secondary["mass"]
+    m1 = primary.latest["mass"]
```

Comment:
@sgossage I don't fully understand where this `latest` comes from. Could you add a doc-string to the function to explain what the input parameter and type is?

And where this equation comes from?